website/pages/docs/concepts/policies.mdx

```diff
@@ -173,8 +173,22 @@ capabilities, which controls a token's access to credentials in Vault.
 ~> Policy paths are matched using the **most specific path match**. This may be
 an exact match or the longest-prefix match of a glob. This means if you define a
 policy for `"secret/foo*"`, the policy would also match `"secret/foobar"`.
-
-!> The glob character referred to in this documentation is the asterisk (`*`). It _is not a regular expression_ and is only supported **as the last character of the path**!
+Specifically, when there are potentially multiple matching policy paths, `P1` and
+`P2`, the following matching criteria is applied:
+
+1. If the first wildcard (`+`) or glob (`*`) occurs earlier in `P1`, `P1` is lower priority
+2. If `P1` ends in `*` and `P2` doesn't, `P1` is lower priority
+3. If `P1` has more `+` (wildcard) segments, `P1` is lower priority
+4. If `P1` is shorter, it is lower priority
+5. If `P1` is smaller lexicographically, it is lower priority
+
+For example, given the two paths, `"secret/*"` and `"secret/+/+/foo/*"`, the first
+wildcard appears in the same place, both end in `*` and the latter has two wildcard
+segments while the former has zero. So we end at rule (3), and give `"secret/+/+/foo/*"`
+_lower_ priority.
```

Comment:
Maybe each paragraph should be prefixed by ~>?  I don't know the details of the markup language well enough to say whether that would create a single note or multiple notes.

Reply:
Or, I suppose, break off everything past "Specifically, when there are multiple...." into the next paragraph so that only the existing sentence is called out with a highlighted box?

Reply:
You can run `make` in `website/` to get a local preview. This is really useful for checking out formatting and link validity.

Reply:
I don't know either. That's why I simply just put them beneath the `->` section. I've seen this done in other docs as well.